src/ape/__init__.py

```diff
@@ -1,3 +1,7 @@
+import signal
+
+signal.signal(signal.SIGINT, lambda s, f: _sys.exit(130))
```

Comment:
Have to try this locally...

Reply:
I am wondering if this a better place to register this.
I generally don't put them here but more like in the `_cli.py` but this file actually takes a long time to load.

Reply:
`_cli.py` might make more sense. that's the actual entry point for running this code I believe

Reply:
For some reason, lol, `__init__` happens first.
I put print statements at the top of both modules and run `ape`.  The `__init__.py` one ran first.  Is this unexpected? Kind of interesting.

Reply:
hmm, entry points must load the module first. weird.